crates/vm-superio-ser/Cargo.toml

```diff
@@ -10,9 +10,9 @@ license = "Apache-2.0 OR BSD-3-Clause"
 edition = "2018"
 
 [dependencies]
-serde = { version = ">=1.0.27", features = ["derive"] }
-versionize = ">=0.1.6"
-versionize_derive = ">=0.1.3"
+serde = { version = "1.0.27", features = ["derive"] }
```

Comment:
nit: since we have multiple crates in the workspace it makes sense to specify in the commit message for which crate this update applies.

Reply:
This change happened in #84 

Will keep in mind for the future though